### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by fatihkahveci**

### CODE REVIEW

File: Resources/translations/time.tr.xliff [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                <source>diff.in.second</source>
+                <target>yaklaşık 1 saniye|yaklaşık %count% saniye</target>
+            </trans-unit>
+            <trans-unit id="9">
+                <source>diff.in.hour</source>
+                <target>yaklaşık 1 saat|yaklaşık %count% saat</target>
+            </trans-unit>
+            <trans-unit id="10">
+                <source>diff.in.minute</source>
+                <target>yaklaşık 1 dakika|in %count% dakika</target>
```
Comment (imanalopher) [inline Resources/translations/time.tr.xliff]: There is you must remove "in" word.
Burda "in" sozune gerek yok.

Comment (weaverryan) [inline Resources/translations/time.tr.xliff]: @imanalopher can you tell me if this translation is better than this other translation? https://github.com/KnpLabs/KnpTimeBundle/pull/76/files#diff-892618d3099db22dd8ec160b4a0b70b5

There are some differences - I'd like to know which are the better translations :).

Thanks!

Comment (fatihkahveci) [inline Resources/translations/time.tr.xliff]: Other one better then from mine 👍

Comment (weaverryan) [inline Resources/translations/time.tr.xliff]: Thanks @fatihkahveci! Then I'll merge the other (once the other PR is ready).

**Pull request closed without merging**